### PR TITLE
Why minus not plus2

### PR DESCRIPTION
**Pull request by fluffy**

### CODE REVIEW

File: draft-ietf-core-senml.md [hunk 1]
```diff
@@ -1299,18 +1299,33 @@ media types for the same formats of the streaming use (SensML). Clipboard
 formats are defined for the JSON and XML form of lists but do not make
 sense for streams or other formats.
 
+When considering if the Media Type should have a "+" or "-" in the
+middle of it, one key issue is that SenML defines it's own fragment
+identifiers and does not use the fragment identifiers that might be
+used by other media types that end in "+xml" or other plus syntax. The
+reason that SenML defines it's own fragment identifiers is so that the
+specific fragments being identified are the same SenML record
```
Comment (akeranen) [inline draft-ietf-core-senml.md:1307]: s/record/Record(s)/

File: draft-ietf-core-senml.md [hunk 2]
```diff
@@ -1299,18 +1299,33 @@ media types for the same formats of the streaming use (SensML). Clipboard
...
+regardless of which media type is used. For example, in the URL
+"coap://example.com/temp#rec=3", there is no way to know from looking
+at the URL if the data would be sent from client to server in
+senml-json or senml-cbor but it is clearly desirable that the same
+record be identified regardless of which media-type was used. This
+would not work for formats such as "+xml" where the fragment
+identifier is defined in a different way thus this specification uses
```
Comment (akeranen) [inline draft-ietf-core-senml.md:1314]: ";" or even a full stop before thus?

**Review by akeranen — APPROVED** [review]
Couple of nits but LGTM

File: draft-ietf-core-senml.md [hunk 3]
```diff
@@ -1299,18 +1299,33 @@ media types for the same formats of the streaming use (SensML). Clipboard
 formats are defined for the JSON and XML form of lists but do not make
 sense for streams or other formats.
 
+When considering if the Media Type should have a "+" or "-" in the
+middle of it, one key issue is that SenML defines it's own fragment
```
Comment (cabo) [inline draft-ietf-core-senml.md:1303]: s/it's/its/ twice

File: draft-ietf-core-senml.md [hunk 4]
```diff
@@ -1299,18 +1299,33 @@ media types for the same formats of the streaming use (SensML). Clipboard
...
+identifiers and does not use the fragment identifiers that might be
+used by other media types that end in "+xml" or other plus syntax. The
```
Comment (cabo) [inline draft-ietf-core-senml.md:1305]: not sure I like "plus syntax".  Stick with "structured syntax suffix"?

File: draft-ietf-core-senml.md (hunk 1 above)
Comment (cabo) [reply draft-ietf-core-senml.md:1307]: s/are/point to/?

**Review by cabo — COMMENTED** [review]
the question I haven't answered for me  is whether we can't go for +json and +cbor, as these are no (yet!?) contaminating SenML with their own fragment identifier syntax.

**Comment by cabo** [comment]
This is now superseded by #115 and #117, which appear to be the smaller change.

**Pull request closed without merging**